core/src/test/java/org/elasticsearch/search/suggest/AbstractSuggestionBuilderTestCase.java

```diff
@@ -166,7 +168,8 @@ protected SB serializedCopy(SB original) throws IOException {
         }
     }
 
-    protected static <T> void maybeSet(Consumer<T> consumer, T value) {
+    // TODO: move this to ESTestCase?
```

Comment:
+1 but maybe this helper so far is limited to the query / suggester tests only. But I think its general enough.